app/blueprints/self_serve/routes.py

```diff
@@ -147,22 +136,42 @@ def build_section():
                 "hover_info": {"title": f["builder_display_name"], "pages": f["pages"]},
             }
         )
-    return render_template("build_section.html", available_forms=available_forms, form=form)
-
+        # save to db here
+    return render_template("create_section.html", available_forms=available_forms, form=form)
 
-@self_serve_bp.route("/add_question", methods=["GET", "POST"])
-def add_question():
-    form = QuestionForm()
-    question = form.as_dict()
+@self_serve_bp.route("/create_form", methods=["GET", "POST"])
```

Comment:
cant we have a open api specification for the endpoints ?

Reply:
i think this can be done as part of the frontend ticket